docs/en/operations/system-tables/replicas.md

```diff
@@ -62,13 +62,13 @@ Columns:
     Note that writes can be performed to any replica that is available and has a session in ZK, regardless of whether it is a leader.
```

Comment:
What about `ZK`? We use ClickHouse Keeper everywhere so ZK could be confusing now?

Reply:
Good catch, I missed that.  I will search through and update.